accounts-db/src/accounts_hash.rs

```diff
@@ -1243,7 +1243,7 @@ pub enum ZeroLamportAccounts {
 
 /// Hash of an account
 #[repr(transparent)]
-#[derive(Debug, Copy, Clone, Eq, PartialEq, Pod, Zeroable)]
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Pod, Zeroable, AbiExample)]
```

Comment:
I guess this is caused by the addition of the rewrites on `Bank`?

Reply:
yeah.